### PR TITLE
fix: csv parser escapes special characters

### LINKED ISSUE #1189: CSV uploader fails to parse fieldKeys with spaces
CSV uploader fails to upload NOAA data because of parsing issue.

Download NOAA data from here: https://docs.influxdata.com/influxdb/v2.0/reference/sample-data/

Attempt to upload while monitoring network requests in dev console. 3 out of the 6 requests fail with 

```
unable to parse 'h2o_feet,location=santa_monica level description="between 6 and 9 feet" 1567365840000000000': invalid field format
```

Looks like we need quotes around "level description" so that it's `"level description"="between 6 and 9 feet"` or escape the space and have it be like `level\ description="between 6 and 9 feet"`

image

### PR DESCRIPTION
**Pull request by genehynson**
Closes #1189

Escapes special characters according to https://docs.influxdata.com/influxdb/cloud/reference/syntax/line-protocol/#special-characters

Adds special characters in valid csv used for e2e tests so we can test these scenarios.

### CODE REVIEW

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 58 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 93 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?vars%5Blighthouse_score_branch%5D=1246)

**Review by Asalem1 — APPROVED** [review]
Legendary. Seriously. I feel like this implementation has gotten way crazier than expected with all the regex parsing and I really hope we can setup a node processing server to stream the data in rather than parsing and writing it to the API in chunks. Until then this is great. Thanks again!

File: cypress/e2e/cloud/annotations.test.ts
Comment (genehynson) [inline cypress/e2e/cloud/annotations.test.ts:251]: This fixes a flaky annotations test

File: cypress/e2e/cloud/annotations.test.ts
Comment (genehynson) [inline cypress/e2e/cloud/annotations.test.ts:65]: Also fixes flaky annotations test

**Comment by genehynson** [comment]
This PR is blocked on https://github.com/influxdata/k8s-idpe/issues/2992

**Comment by Asalem1** [comment]
@genehynson this still blocked?

**Comment by genehynson** [comment]
> @genehynson this still blocked?

Yeah I can't debug the k8s-idpe test failure because I can't run the cypress tests locally with k8s-idpe

**Comment by genehynson** [comment]
I removed the fixes to the flaky annotations tests and moved them to their own PR to unblock those fixes. https://github.com/influxdata/ui/pull/1312

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 56 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](true1246)

**Pull request merged**
